assets/src/edit-story/components/library/panes/media/dropDownMenu.js

```diff
@@ -39,6 +39,9 @@ const MoreButton = styled(More)`
   position: absolute;
   top: 8px;
   right: 8px;
+  background: ${({ theme }) => theme.colors.bg.v4};
```

Comment:
Just to confirm: should the Media edit icon in the Accessibility panel also get a new background color or is it intended that it's only for the Media menu?
<img width="254" alt="Screenshot 2020-06-25 at 15 53 55" src="https://user-images.githubusercontent.com/3294597/85732988-1cd5aa00-b6fc-11ea-9eb5-ee44f9485779.png">

(This icon will change within #1218 anyway though so perhaps it's ok to keep as is.)

Reply:
@miina Yes the Media edit icon in the Accessibility panel should also get a new background color

Reply:
Done 👍